### PR TITLE
added workflow to publish site generated by jekyll 

### PR DESCRIPTION
**Pull request by PlutonicJ**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+git submodule init
+git submodule update
+
+echo "#################################################"
```
Comment (BillRaymond) [inline entrypoint.sh:7]: It looks like you are removing the readme and .gitignore. Is that for the published site or on my code repo? If it is on my GitHub code repo, I do not want to delete those.

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 0 * * *'
```
Comment (BillRaymond) [inline .github/workflows/push.yml:6]: I would like the code to run any time there is a new commit or push of code up from my local computer

**Review by BillRaymond — CHANGES_REQUESTED** [review]
Please take a look at the two comments I added

File: entrypoint.sh (hunk 1 above)
Comment (PlutonicJ) [reply entrypoint.sh:7]: The files aren't removed on any repository. If I don't remove them, they are pushed to the target repository, where they aren't necessary.

File: .github/workflows/push.yml (hunk 2 above)
Comment (PlutonicJ) [reply .github/workflows/push.yml:6]: Every push to the main branch triggers the build. Additionally it is triggered at 00:00.

Comment (BillRaymond) [reply .github/workflows/push.yml:6]: Ah, okay. I had seen the comment that the code runs manually and made a broad assumption :-)

Comment (BillRaymond) [reply .github/workflows/push.yml:6]: Looks like your comments point to everything being okay, so I merged the code and will test.

Comment (PlutonicJ) [reply .github/workflows/push.yml:6]: I have created another pull request as the target repository has been mine and not yours. Additionally I added some comments.

**Pull request merged**